portal/tests/test_independent_student.py

```diff
@@ -249,36 +249,6 @@ def test_join_class_revoked(self):
 
         assert is_indep_student_join_request_revoked_message_showing(selenium)
 
-    def test_join_class_accepted(self):
```

Comment:
??

Reply:
Test failing and it's too much debt to keep trying to fix it. We can't figure it out and debugging it is a nightmare (selenium, hey). 

We have two more tests that test this redirection functionality that are still in the code.

Reply:
Travis logs get cut off so can't see tracebacks, can't run selenium tests individually locally and get it to fail